tests/tests/cpld_bmark_p8

```diff
@@ -140,3 +151,5 @@ export RST_END=$RUN_END
 export RST_2_END=$RUN_END
 
 export FV3_RUN=cpld_control_run.IN
+export FIELD_TABLE=field_table_thompson_noaero_tke_GOCART
+export DIAG_TABLE=diag_table_p7.2_template
```

Comment:
Is this the default DIAG_TABLE for all cpld_*p8 tests? If only the remaining P7 tests are using the older diag_table_template, maybe the p7.2_template should be made the default.

Reply:
done